jiant/models.py

```diff
@@ -541,6 +541,9 @@ def build_task_specific_modules(task, model, d_sent, d_emb, vocab, embedder, arg
             params=task_params,
         )
         setattr(model, "%s_mdl" % task.name, module)
+    elif isinstance(task, SentenceOrderTask):
```

Comment:
Is there a reason this isn't a PairClassificationTask? In any case, it probably makes more sense readability-wise to cluster this with the single-purpose task classes below, rather than with the broad classes here. (Just proposing to rearrange the else clauses, not change behavior.)

Reply:
Ah, I see that this is a PairClassificationTask, but it's the only one that has to preempt the generic build_sentence_pair_module, so it has to be here. This seems like it's making an already messy pile of if-elses a bit harder to follow. I don't see any obvious alternative that doesn't involve a larger refactor, but it'd be good to get @pyeres's input before a merge.